scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/UniversalSchedulerListener.java

```diff
@@ -70,36 +70,64 @@ public UniversalSchedulerListener() {
         this.internalListener = null;
     }
 
-    /* (non-Javadoc)
-     * @see org.ow2.proactive.scheduler.common.SchedulerEventListener#jobStateUpdatedEvent(org.ow2.proactive.scheduler.common.NotificationData)
+    /*
+     * (non-Javadoc)
+     * 
+     * @see org.ow2.proactive.scheduler.common.SchedulerEventListener#jobStateUpdatedEvent(org.ow2.
+     * proactive.scheduler.common.NotificationData)
      */
     public void jobStateUpdatedEvent(NotificationData<JobInfo> notification) {
         this.internalListener.jobStateUpdatedEvent(notification);
     }
 
-    /* (non-Javadoc)
-     * @see org.ow2.proactive.scheduler.common.SchedulerEventListener#jobSubmittedEvent(org.ow2.proactive.scheduler.common.job.JobState)
+    /*
+     * (non-Javadoc)
+     * 
+     * @see org.ow2.proactive.scheduler.common.SchedulerEventListener#jobStateUpdatedEvent(org.ow2.
+     * proactive.scheduler.common.NotificationData)
+     */
+    public void jobUpdatedFullDataEvent(JobState job) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)